pkg/kube/inject/inject.go

```diff
@@ -763,10 +764,11 @@ func IntoObject(sidecarTemplate string, valuesConfig string, meshconfig *meshcon
 	}
 
 	metadata.Annotations[annotation.SidecarStatus.Name] = status
+	if metadata.Labels == nil {
+		metadata.Labels = make(map[string]string)
+	}
+	metadata.Labels[model.RevisionLabel] = os.Getenv("REVISION")
```

Comment:
We should not use `os.Getenv()`. There is supposed to be a linter for this so if its not failing that must be broken. Can you reference `revisionVar` in pilot/pkg/bootstrap (and make it `RevisionVar`)?